src/compiler/nvcc.rs

```diff
@@ -118,6 +118,7 @@ impl CCompilerImpl for NVCC {
         //NVCC only supports `-E` when it comes after preprocessor
         //and common flags.
         cmd.arg("-E")
+            .arg("-D__FILE__=\"empty\"")
```

Comment:
I don't quite follow, why do we need to "get rid of line info from nvcc"?
I'm _guessing_ that it's another potential improvement with its own merits - perhaps it should be split into a separate patch?